src/batch/normalization/normalization.spec.ts

```diff
@@ -3,69 +3,128 @@ import { MockUtils } from '../../shared/infrastructure/utils/mock.utils'
 jest.mock('./services/saveToMongo')
 const mockUtils = new MockUtils()
 const fakeMetadonnees = mockUtils.metadonneesDtoMock
-jest.mock('./services/extractMetadonneesFromS3', () => ({
-  extractMetadonneesFromS3: jest.fn(() => fakeMetadonnees)
-}))
+
 jest.spyOn(process, 'exit').mockImplementation()
 
 import { normalizationJob } from './normalization'
+import * as fetchDecisionListFromS3 from './services/fetchDecisionListFromS3'
```

Comment:
est-on obligé d'import * ? Faire in import {... } ne fonctionne pas ?

Reply:
Ajouter dans le convention.md : 
- Si on mock une fonction : import * as ... 
- Si on mock une class : import { ... } puis maClasse.prototype lors du jest.mock()